packages/quantic/cypress/e2e/default-2/generatedAnswer/generated-answer.cypress.ts

```diff
@@ -52,144 +58,104 @@ const genQaMessageTypePayload = {
 };
 
 describe('quantic-generated-answer', () => {
+  beforeEach(() => {
+    cy.clock(0, ['Date']);
```

Comment:
why do we need this? To wait for the answer to be generated fully?

Reply:
No, this is for the test where we will hover over a citation, in order to log analytics event we need to hover over the citation for one second or more.

So to win time and avoid making our test to wait, I'm playing with the clock using this Cypress functionality, so now no need to wait we will fast forward time at our convenience.

Reply:
ok cool!